app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -53,16 +68,27 @@ protected void onCreate(Bundle savedInstanceState) {
         expListView.setAdapter(listAdapter);
     }
 
-    public static void swapFragmentIn(FragmentActivity activity, Fragment fragment, String TAG)
+    public static void swapFragmentIn(FragmentActivity activity, Fragment fragment, String TAG, boolean addToBackStack)
```

Comment:
One change made to the function which was written for all fragment transactions. In some cases, it was not necessary to add it to backstack, hence I made this small change where I send boolean value along with the function call